frontends/web/src/components/view/view.tsx

```diff
@@ -85,9 +85,13 @@ export const View = ({
       )}
       {withBottomBar && (
         <div style={{ marginTop: 'auto' }}>
-          <Footer>
+          <footer className={style.footer}>
             <SwissMadeOpenSource />
-          </Footer>
+            <div className="m-right-half hide-on-small">
+              <Version />
+            </div>
+            <LanguageSwitch />
+          </footer>
```

Comment:
Hm, not rly a big deal but instead of sort of duplicating `Footer` here, would you think it makes sense to just re-style the `Footer` (to not use the inline `flex`, `flex-row`, `flex-end`,etc)? 

As it seems like this is the only difference between the `Footer` component and what we have above? 🤔

Reply:
yes I thought about this, but then on the other hand `<Footer>` is only used in theses components, which I think all should be migrated to `<View>`

![Screen Shot 2022-12-19 at 08 19 32](https://user-images.githubusercontent.com/546900/208369478-be087c3b-2a5e-49d9-bcd9-40108c71bfa4.png)

Reply:
Mhm, okay. So once we migrated all of these to be using the `View` components, we can remove `Footer` completely? And maybe if it's relevant enough, should we think of a way to pass children into the footer (if it's inside `View`)?

Reply:
The view component could export `<ViewFooter>`, but right now it is not needed anywhere.